dags/sources/tasks/business_logic/db_write_suggestion.py

```diff
@@ -0,0 +1,95 @@
+import json
+import logging
+from datetime import datetime
+
+import pandas as pd
+from shared.tasks.database_logic.db_manager import PostgresConnectionManager
+from sources.config import shared_constants as constants
+
+logger = logging.getLogger(__name__)
+
+
+def db_write_suggestion(
```

Comment:
```suggestion
def db_write_suggestion_per_action_type(
```

? à voir si c'est nécessaire ou pas